docs/installation.md

```diff
@@ -19,8 +22,9 @@ Test that this worked:
 
     > smk help
 
-You should see the help information for `smk-cli`.
+You should see the help information for `smk-cli`. If you don't and are on Windows, then it's likely your npm folder (`%APPDATA%\npm`) is not part of the system PATH variable. The simplest way to add it is with the [GUI](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/). After adding the npm folder to path, `> smk help` should work.
```

Comment:
Nit: "on Windows" - maybe "using Windows"? 

For the link - "GUI" is good but I think we could be more specific - maybe "Control Panel"? And I would advise using a more authoritative link as this one is more likely to go away (if the author deletes his blog, say) - I'd recommend a link from Microsoft.

I'd remove the '>' (prompt) with 'smk help' here - I think the prompt works well when the cmd is its own line but looks a bit awkward to me inline like this.

Reply:
Found a slightly more official page in Oracle docs. Nothing easily found on MS oddly.

Reply:
What do you think of this link instead? https://www.computerhope.com/issues/ch000549.htm#windows10 The Oracle link has content specific to installing its tool; this is more generic.

Reply:
Works for me - doesn't have as many images but heck they been around since 1998